server/data/data-api/src/main/java/org/apache/james/rrt/api/RecipientRewriteTable.java

```diff
@@ -122,6 +122,8 @@ default Mappings getResolvedMappings(String user, Domain domain) throws ErrorMap
      */
     Map<MappingSource, Mappings> getAllMappings() throws RecipientRewriteTableException;
 
+    int getMappingLimit();
```

Comment:
boolean recursiveResolutionEnabled();

BTW exposing these parameters is a code smell about duplicated logic IMO

Reply:
My bad understanding of the expected behaviour of this method is a clear sign that javadoc is needed ;-)

Reply:
I do not need it;

And not duplicated code, misplaced maybe.

Reply:
Im still unconfortable leaking the conf in the interface here. Let me suggest:

 1. Add a pojo for abstract rrt conf
 2. Parse and inject it via guice, workaround with spring
 3. Reuse the exact same pojo in CanSendFromImpl.

Reply:
I have been tracking the code in my IDE, there is no usage for `setMappingLimit`

Please don't leak the configuration and rely on a POJO.